setup.cfg

```diff
@@ -0,0 +1,38 @@
+[tool:pytest]
+addopts = --strict-markers
+norecursedirs = .* build dist CVS _darcs {arch} *.egg venv env virtualenv
+
+
+[mypy]
+files = commitizen, tests
```

Comment:
Do we need to enforce typehints in tests?

Reply:
It can enhance the correctness of our test cases. It does not require us to annotate type in tests, but ask us to use the correct type.